modules/library/src/main/scala/zio/elasticsearch/Boost.scala

```diff
@@ -16,8 +16,17 @@
 
 package zio.elasticsearch
 
-import zio.elasticsearch.ElasticQuery.{ElasticPrimitive, MatchAllQuery, TermQuery, WildcardQuery}
-import zio.elasticsearch.ElasticQueryType.{MatchAll, Term, Wildcard}
+import zio.elasticsearch.ElasticQuery.{
```

Comment:
I guess we can `import zio.elasticsearch.ElasticQuery._` instead.

Reply:
Changes done !